front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/SingleIdeaWidget/Settings/IdeaFilter.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useMemo } from 'react';
+
+// hooks
+import useIdeas from 'api/ideas/useIdeas';
+
+// components
+import { Box, Select, Spinner, Text } from '@citizenlab/cl2-component-library';
+
+// i18n
+import useLocalize from 'hooks/useLocalize';
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from '../messages';
+
+// typings
+import { IOption } from 'typings';
+
+interface Props {
+  label: string;
+  phaseId: string;
+  ideaId?: string;
+  onIdeaFilter: (filter: IOption) => void;
+}
+
```

Comment:
Hmm, I'm not sure about using a select for this. It will not scale well for projects with a large number of ideas...

We could consider something like a searchable select with pagination / load more mechanism, like we use for the users select, but that will also involve backend work.

What do you think?